sequel-ace.xcodeproj/project.pbxproj

```diff
@@ -3733,6 +3709,7 @@
 		96BC47C72500A84F003C8105 /* Beta */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
Looks like the flag got set to YES for beta and Distribution but not for all of the other configs? Looks like release is set to NO?

Reply:
I believe not. It was set for the whole project, but excluded two targets (xib and sptunnel) - they were throwing warnings